src/sorting/README.md

```diff
@@ -235,3 +241,6 @@ __Properties__
 
 [comb-sort]: https://upload.wikimedia.org/wikipedia/commons/4/46/Comb_sort_demo.gif
 [comb-sort-wiki]: https://en.wikipedia.org/wiki/Comb_sort
+
+[sleep-sort]: <no image>
+[sleep-sort-wiki]https://ja.m.wikipedia.org/wiki/バケットソート#.E3.82.B9.E3.83.AA.E3.83.BC.E3.83.97.E3.82.BD.E3.83.BC.E3.83.88
```

Comment:
Since we could not confirm wikipedia in English, we refer to the section described in Japanese.
(If necessary, I would be willing to translate it into English and create a wikipedia page.)